execute_test.go

```diff
@@ -88,7 +88,10 @@ func TestRunEchoWithChan(t *testing.T) {
 func TestGetExitCode(t *testing.T) {
 	timeoutTime := time.Duration(1) * time.Second
 
-	cmd := exec.Command("grep", "go")
+	//This command creates a parent and child process
+	//so that we can test killing both parent and child
+	//processes
+	cmd := exec.Command("/bin/sh", "-c", "watch date > date.txt")
```

Comment:
What about a command that creates a process that creates a process?  What about a command that creates two processes that each create two processes?  I understand the permutations are infinite but it does seem like we should go at least a few levels deep to cover things like bash -> make -> bash -> postgres -> forked postgres's etc